lib/MusicBrainz/Server/Controller/Attributes.pm

```diff
@@ -96,7 +96,7 @@ sub attribute_index : Chained('attribute_base') PathPart('') {
     );
 }
 
-sub create : Chained('attribute_base') RequireAuth(account_admin) SecureForm {
+sub create : Chained('attribute_base') RequireAuth(relationship_editor) SecureForm {
```

Comment:
While it is true that it doesn’t have anything to do with accounts, modifying “attributes” (or whatever their names) is not as safe as modifying relationship types at the moment: It doesn’t create edits, there are little (if any) database constraints, there is little (if any) form validation.

So it currently mostly has to do with not breaking the website at a large scale.

Editing those should probably rather require dedicated privilege flags, so should editing genre and editing instrument, to follow the existing model of dedicated privileges for editing area.

Reply:
> Editing those should probably rather require dedicated privilege flags

It feels these are conceptually very close to relationships (basically it's "schema editor") but I see the point about safety measures. I'm fine with having a separate option if you prefer that, shouldn't be hard to implement a new `privs` flag (I understand those do not require schema changes anyway) - no idea what we'd call it though!